php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2116,7 +2102,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
 
         // Loop over this instance data
         foreach ($thisData AS $key=>$value) {
-            if (!in_array($key, $this->_doubleDataEntryDiffIgnoreColumns)) {
+            if (!in_array($key, $this->doubleDataEntryDiffIgnoreColumns)) {
```

Comment:
doubleDataEntryDiffIgnoreColumns should be _doubleDataEntryDiffIgnoreColumns

Reply:
thanks, I missed that on re-review. I just created a ticket.